importers/replay_comment_importer.py

```diff
@@ -27,6 +27,8 @@ def can_load_this_file(self, file_contents):
         return True
 
     def _load_this_line(self, data_store, line_number, line, datafile, change_id):
+        if line.text == "":
```

Comment:
I _think_ you may be able to `strip()` the `line.text` before you do this test.

We should skip the line if it contains a couple of spaces, or a tab.

Reply:
Done:
https://github.com/debrief/pepys-import/pull/411/commits/ffc17fa986e6084a19c037b931e69822eab3098f